preview-src/troubleshooting.adoc

```diff
@@ -0,0 +1,126 @@
+= How to write a clean troubleshooting section
```

Comment:
Is the content is on purpose? To me, the Troubleshooting exampes from misc.adoc are better...